mllib/src/test/java/org/apache/spark/mllib/clustering/JavaLDASuite.java

```diff
@@ -126,8 +128,12 @@ public Boolean call(Tuple2<Long, Vector> tuple2) {
     assertEquals(topicDistributions.count(), nonEmptyCorpus.count());
 
     // Check: javaTopTopicsPerDocuments
-    JavaRDD<scala.Tuple3<java.lang.Long, int[], java.lang.Double[]>> topTopics =
-      model.javaTopTopicsPerDocument(3);
```

Comment:
This check is not sufficient due to runtime type erasure. We need to pick out the elements.